catalogue/webapp/services/wellcome/catalogue/images.ts

```diff
@@ -50,6 +74,14 @@ export async function getImages(
   const extendedParams = {
     ...params,
     ...query,
+    'source.production.dates.from': toIsoDateString(
+      query['source.production.dates.from'] as string,
```

Comment:
Bit annoying that this isn't properly typed (I know it's the same in the existing works code) but think that's a problem for another day